testing/references/kpis/cost_dict_SingleZone.csv

```diff
@@ -0,0 +1,5 @@
+keys,value
```

Comment:
Same comment as in `cost_dict_MultiZone.csv`.

Reply:
This reference was new when I ran the tests locally.  I think it is from ``test_iterative_call``.